UITests/BookmarksAndFavoritesTests.swift

```diff
@@ -95,9 +95,8 @@ class BookmarksAndFavoritesTests: XCTestCase {
         showFavoritesPreferenceToggle = app.checkBoxes["Preferences.AppearanceView.showFavoritesToggle"]
 
         app.launch()
-        resetBookmarks()
```

Comment:
Can we also remove `resetBookmarks()` function and `resetBookMarksMenuItem` property from this file?

Reply:
Sure!